index.html

```diff
@@ -41,7 +41,7 @@ <h3>Short and sweet. No spam.</h3>
     </form>
     <label id="subscribed">Subscribed. Check your inbox.</label>
     <div class="details left-buffer">
-      <a href="/2019-05-21.html">Read latest issue</a>
+      <a href="/archive.html">Read past issues</a>
```

Comment:
I prefer to show only the last one under the subscription email

Reply:
Reverted, please check now.

Reply:
@siddharthkp I reverted it to show only last issue. Now, the only changes are to the archive page.